test/test_data/__init__.py

```diff
@@ -16,6 +16,7 @@
     "dnsmasq",
     "dovecot",
     "dpkg",
+    "enscript",
```

Comment:
I notice you have to add the checker in two __init__.py files.  Is there any way we could make it so we only have one canonical list of checkers?

Reply:
I filed issue #1223 to look into this later, since it's not something that needs to be fixed in this checker specifically.